engines/asylum/system/text.cpp

```diff
@@ -135,10 +163,32 @@ void Text::drawChar(char character) {
 	_position.x += (int16)fontLetter->surface.w + fontLetter->x - _curFontFlags;
 }
 
+void Text::drawChinese(const Common::U32String &utext) {
+	loadChineseFont();
+	if (!_chineseFont)
+		return;
+	Graphics::Surface *surf = getScreen()->getSurface();
+	uint8 color = 0;
+	switch (_fontResource->getResourceId()) {
+	default:
```

Comment:
Is it some leftover? There is only the default case covered.

Reply:
Quite the opposite: not yet implemented feature. Added a TODO for this